lapis/cmd/actions/annotate.moon

```diff
@@ -17,7 +17,7 @@ build_command = (cmd, config) ->
   command = { cmd }
 
   if password = config.postgres.password
-    table.insert 1, command, "PGPASSWORD=#{shell_escape password}"
+    table.insert command, 1, "PGPASSWORD=#{shell_escape password}"
```

Comment:
Probably should have made this a separate PR (happy to, if helpful) but this seems like a genuine typo. :)